### PR TITLE
Fix: unset ARTIFACTS in maistra multicluster service mesh integ test 

### PR DESCRIPTION
**Pull request by yxun**

### CODE REVIEW

**Comment by yxun** [comment]
/pj-rehearse pull-ci-maistra-istio-maistra-2.4-integ-servicemesh-multicluster-2-4

**Comment by yxun** [comment]
/pj-rehearse pull-ci-maistra-istio-maistra-2.4-integ-servicemesh-multicluster-2-4

File: ci-operator/config/maistra/istio/maistra-istio-maistra-2.2.yaml [hunk 1]
```diff
@@ -313,13 +313,15 @@ tests:
...
+        # temporary workaround: unset ARTIFACTS env for integ-servicemesh-multicluster
         oc rsh -n "${MAISTRA_NAMESPACE}" "${MAISTRA_MC_POD}" \
+          bash -c \
+          "unset ARTIFACTS; \
           entrypoint \
           prow/integ-suite-kind.sh \
           --topology \
           MULTICLUSTER \
-          test.integration.servicemesh.kube
-        oc cp "${MAISTRA_NAMESPACE}"/"${MAISTRA_MC_POD}":"${ARTIFACT_DIR}"/. "${ARTIFACT_DIR}"
```
Comment (jwendell) [inline ci-operator/config/maistra/istio/maistra-istio-maistra-2.2.yaml:322]: Why did you remove the `oc cp` command?

Comment (yxun) [reply ci-operator/config/maistra/istio/maistra-istio-maistra-2.2.yaml:322]: I added that line back in the second commit
